### PR TITLE
Linting + js sugar

### PR DESCRIPTION
**Pull request by san983**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -19,7 +18,8 @@
       "32": "images/logo/32.png",
       "48": "images/logo/48.png",
       "128": "images/logo/128.png"
-    }
+    },
+    "default_title": "DNSimple"
```
Comment (OleMchls) [inline manifest.json:23]: What does this option do? It's not documented here: https://developer.chrome.com/apps/manifest

File: manifest.json [hunk 2]
```diff
@@ -6,8 +6,7 @@
   "homepage_url": "https://dnsimple.com",
   "description": "Always know if it resolves awesome",
   "permissions": [
-    "history",
-    "https://isitdnsimple.com/"
```
Comment (OleMchls) [inline manifest.json]: You can't remove this, as this allows the extensions to make requests to that domain (which is used to determine the status).

**Review by OleMchls — CHANGES_REQUESTED** [review]
You need the `https://isitdnsimple.com/`permission.

Thanks for taking the code after into the 2010s :D

File: manifest.json (hunk 1 above)
Comment (san983) [reply manifest.json:23]: Just a tooltip when you get over the button :) More info here ➡️ https://developer.chrome.com/extensions/pageAction

File: manifest.json (hunk 2 above)
Comment (san983) [reply manifest.json]: Are you sure about that? it's working good locally for me w/o that. Also I was not able to find that in the docs https://developer.chrome.com/apps/declare_permissions

Comment (OleMchls) [reply manifest.json]: Yes 👉  https://developer.chrome.com/extensions/xhr#requesting-permission

Your docs link to ChromeOS

Comment (san983) [reply manifest.json]: You're totally right!

**Pull request merged**